fps-game-jam-org/open-game-jam-2020
# Feature/wind controller

Add wind prefab

Test:
  * Open the WindTest scene and run.
  * Verify that bird prefab contains both a Rigidbody 2D and Box Collider 2D
  * Verify the wind sprite is animated
  * Verify that all bird prefabs have a force applied in the direction of the wind when intersecting the wind prefab
  * Verify that using WASD applies forces to the bird prefab in the desired direction of movement
